### PR TITLE
Make coverage optional

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@ inputs:
   inline:
     description: 'Value passed to the --inline flag. Options: yes | no. Default value: yes.'
     default: 'yes'
+  coverage:
+    description: 'Value determining wether to test with coverage or not. Options: yes | no. Default value: yes.'
```
Comment (fredrikekre) [inline action.yml:14]:
```suggestion
    description: 'Value determining wether to test with coverage or not. Options: true | false. Default value: true.'
```

File: action.yml [hunk 2]
```diff
@@ -10,6 +10,9 @@ inputs:
...
+    default: 'yes'
```
Comment (fredrikekre) [inline action.yml:15]:
```suggestion
    default: 'true'
```

**Review by SaschaMann — APPROVED** [review]
LGTM, thanks!

**Comment by SaschaMann** [comment]
Released as v1.2.0/v1.2 and updated v1/latest tags: https://github.com/julia-actions/julia-runtest/releases/tag/v1.2.0

**Comment by fredrikekre** [comment]
A motivating example: StaticArrays with and without coverage:
```
@time Pkg.test("StaticArrays"; coverage=false)
739.193126

@time Pkg.test("StaticArrays"; coverage=true)
2385.432197
```
Seems like a waste of resources to run with coverage on many workers/OSs when you cover the same code.

**Comment by SaschaMann** [comment]
That's a pretty large difference. Should we make `false` the default and tag a v2 release?

**Comment by fredrikekre** [comment]
I think the difference is really dependent on the package though. I think here it might be because StaticArrays uses some `@generated`.

**Pull request merged**